server/routers/archive.py

```
@@ -0,0 +1,78 @@
+        "service": "Archive Management",
+        "endpoints": {
+            "/trigger": "Manually trigger disaster archiving",
+            "/task/{task_id}": "Get archiving task status",
+        },
+    }
+
+
+@router.post("/trigger")
+def trigger_archive(
+    days_threshold: int = 2,
+    current_admin: User = Depends(get_current_admin)
```

Ruff (0.14.1)

Unused function argument: `current_admin`

(ARG001)